src/components/ManageAccountsDrawer/components/ImportAccounts.tsx

```diff
@@ -45,18 +49,28 @@ type TableRowProps = {
   accountId: AccountId
   accountNumber: number
   asset: Asset
-  toggleAccountId: (accountId: AccountId) => void
+  onAccountIdActiveChange: (accountId: AccountId, isActive: boolean) => void
```

Comment:
Assuming we get the initial state properly propagated (i.e toggled state properly synced), then the previous flavor should work?

Reply:
Also we should probably keep on using the `toggled` terminology regardless, to make the distinction mentioned in https://github.com/shapeshift/web/pull/6780/files#r1588955759 clearer - `active` doesn't say much, the user event here is a toggle

Reply:
good point, ill fix this in a follow-up as it'll introduce conflicts with the next PR